### ISSUE TITLE
run model training using tensorpack 

### ISSUE THREAD

**Issue by zhangyu2ustc** [SIMEXP/fmri_predict#7]
In order to use tensorpack which was based on tensorflow, we need to rewrite the keras-based script. Just using this to build the model:
   from tensorpack.contrib.keras import KerasModel
   from tensorpack.utils.gpu import get_num_gpu
   from tensorflow.python.keras.models import Model

    M = KerasModel(
        resnet50,
        inputs_desc=[InputDesc(tf.uint8, [None, 224, 224, 3], 'images')],
        targets_desc=[InputDesc(tf.float32, [None, 1000], 'labels')],
        input=df_train,
        trainer=SyncMultiGPUTrainerReplicated(get_num_gpu()))

tutorial:
    https://github.com/tensorpack/tensorpack/tree/master/examples/keras

**Comment by zhangyu2ustc** [comment]
New models:
   HCP_task_fmri_cnn_tensorpack_changesize_bk4_wm_test.py
